vendor/github.com/opencontainers/image-spec/identity/chainid_test.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2016 The Linux Foundation
```

Comment:
I don't think godep vendors test.

Could you try godeps or manually remove this file?